Source/WebCore/platform/graphics/FontCascade.h

```diff
@@ -319,10 +321,16 @@ class FontCascade : public CanMakeWeakPtr<FontCascade> {
         return advancedTextRenderingMode();
     }
 
+    struct Spacing {
+        Length letter;
+        Length word;
+        Spacing() : letter(LengthType::Fixed) , word(LengthType::Fixed) { };
+        bool operator==(const Spacing& other) const { return letter == other.letter && word == other.word; }
```

Comment:
The default operator should give you this.

Reply:
That's what I thought, but it wouldn't compile without it. :/ Idk what to do.